js/web/web-backend.js

```diff
@@ -162,3 +164,21 @@ async function workerCall(worker, type, message) {
   worker.postMessage({ type, id, message });
   return (await promise).result;
 }
+
+function allocateWasmMemoryForUserAgent(userAgent) {
+  const isIOSDevice = /iPad|iPhone|iPod/.test(userAgent);
+  const isSafari = /Safari/.test(userAgent);
+  if (isIOSDevice && isSafari) {
```

Comment:
IIRC, the OOM occurred for all user agents on iOS (they all use the same Webkit engine under the hood), so should we remove the second condition?

Reply:
I think the `Safari` will be present in userAgents anyway.